### PR TITLE
input and table bugs are fixed

### PR DESCRIPTION
**Pull request by kamal-kishor**
fixing all pointed bugs

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -15,15 +15,17 @@ export const Button: React.FunctionComponent<ButtonProps> = (props) => {
     let _style: React.CSSProperties = style || {};
     let _className: string = 'commonButton';
     let _type: string = 'submit';
-    /* Override defaults */
+    let _children: any = 'Button';
```
Comment (expertdeveloper5821) [inline src/components/Button/Button.tsx]: write proper types. Don't use 'any'

**Comment by kamal-kishor** [comment]
Fixing all isues on table input textarea and button component

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    presets: [['@babel/preset-env', { targets: { node: 'current' } }], '@babel/preset-typescript']
+};
```
Comment (expertdeveloper5821) [inline babel.config.js:3]: why we have added babel file in our package and what's the use for this.

**Comment by kamal-kishor** [comment]
fixing issues over Table component

File: src/components/Table/Table.tsx [hunk 3]
```diff
@@ -4,25 +4,34 @@ import TableRow from './TableRow';
...
+    id: string;
+    title: string;
+    width?: number;
+    type: string;
+    value: string;
+}
 
 type TableProps<T> = {
-    data: T[];
+    data: any[];
```
Comment (expertdeveloper5821) [inline src/components/Table/Table.tsx]: why we have any here

File: src/components/Table/TableRowCell.tsx [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
     item: T;
     column: IColumnType<T>;
-    onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+    buttons?: IButtonType<T>;
+    onClick?: (event: React.MouseEvent<HTMLButtonElement>, rowData: T) => void;
 }
 
-const TableRowCell = <T extends Props<T>>({ item, column, onClick }: any): JSX.Element => {
-    const value = get(item, column.key);
+const TableRowCell = <T extends Props<T>>({ item, column, buttons, onClick }: any): JSX.Element => {
```
Comment (expertdeveloper5821) [inline src/components/Table/TableRowCell.tsx]: we have any here also

File: src/components/Select/Select.tsx [hunk 5]
```diff
@@ -30,51 +30,14 @@ export interface itemType {
...
-    };
-
-    // Here getOptions is used to configure the options passed through props
-    const getOptions = () => {
-        if (!searchValue) {
-            return options;
-        }
-        return options.filter((option: any) => option.toLowerCase().indexOf(searchValue.toLowerCase()) >= 0);
-    };
+    const [selectedValue, setSelectedValue] = useState<any>(isMulti ? [] : '');
```
Comment (expertdeveloper5821) [inline src/components/Select/Select.tsx]: please remove any from here

**Pull request merged**